setup.py

```diff
@@ -1,4 +1,7 @@
 # -*- coding: utf-8 -*-
+raise DeprecationWarning('Use rez-build to build this package at Rodeo.')
```

Comment:
The raise is here a safeguard against human error that could break production. 

The release of the package requires having the symlink to be updated. A pip/rez-pip would not do the trick.

I understand that the raise might ask some (very minor) additional work when rez-pip will be reviewed. But I would prefer to protect production stability.